source/_patterns/02-molecules/breadcrumb/_breadcrumb.scss

```diff
@@ -0,0 +1,18 @@
+// All component Sass needs non-printing base config
```

Comment:
Let's make this an atom. Even though it renders other things, it doesn't necessarily include any other Patterns.